src/postgres/tree/PostgresStoredProceduresTreeItem.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ClientConfig } from 'pg';
+import { TreeItemIconPath } from "vscode-azureextensionui";
+import { getThemedIconPath } from "../../constants";
+import { getPostgresProcedureQueryRows } from '../getPostgresProcedureQueryRows';
+import { IPostgresProceduresQueryRow } from '../IPostgresProceduresQueryRow';
+import { PostgresDatabaseTreeItem } from './PostgresDatabaseTreeItem';
+import { PostgresResourcesTreeItemBase } from './PostgresResourcesTreeItemBase';
+import { PostgresStoredProcedureTreeItem } from './PostgresStoredProcedureTreeItem';
+
+export class PostgresStoredProceduresTreeItem extends PostgresResourcesTreeItemBase {
+    public static contextValue: string = 'postgresStoredProcedures';
+    public readonly contextValue: string = PostgresStoredProceduresTreeItem.contextValue;
+    public readonly label: string = 'Stored Procedures';
+    public readonly childTypeLabel: string = 'Stored Procedure';
+
+    constructor(parent: PostgresDatabaseTreeItem, clientConfig: ClientConfig) {
+        super(parent);
+        this.clientConfig = clientConfig;
+    }
+
+    public get iconPath(): TreeItemIconPath {
+        return getThemedIconPath('list-unordered.svg');
+    }
+
+    public hasMoreChildrenImpl(): boolean {
+        return false;
+    }
+
+    public async loadMoreChildrenImpl(): Promise<PostgresStoredProcedureTreeItem[]> {
+        // Adapted from https://aka.ms/AA83fg8
+        const storedProceduresQuery: string = `select n.nspname as schema,
```

Comment:
Looks like only one or two lines of this query are changed from the functions query. Would prefer to share this instead of copying. I think it makes the most sense if it's a part of `getPostgresProcedureQueryRows`. Also, I would move the definition of `IPostgresProceduresQueryRow` into `getPostgresProcedureQueryRows.ts`, since the structure of the interface is completely dependent on how you authored this query. Just makes it easier to understand if you can see the query and the interface side-by-side.